app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons = [Person(p["name"], p["age"]) for p in people]
+
+    for p in people:
+        if "wife" in p and p["wife"] is not None:
+            Person.people[p["name"]].wife = Person.people[p["wife"]]
+
+        if "husband" in p and p["husband"] is not None:
+            Person.people[p["name"]].husband = Person.people[p["husband"]]
+
+    return persons
```

Comment:
The `create_person_list` function should return the `persons` list, which contains the created `Person` objects. Ensure that this list is returned to use the function's output effectively.